web/templates/index.html

```diff
@@ -107,7 +107,7 @@ <h1>botamusique Web Interface</h1>
                         </td>
                     </tr>
                     <tr class="playlist-item-template d-none">
-                        <th scope="row" class="playlist-item-index d-none d-md-table-cell"></th>
+                        <th scope="row" class="playlist-item-index d-none d-md-table-cell">1</th>
```

Comment:
I have added this because this header is empty only when the page has an empty playlist and is reloaded. After emptying the playlist, this is how the header looks. It is more consistent this way, and it also gets rid of a particular error.

Although we might want to hide this with an ARIA tag instead. A hidden header should not be shown to anyone, really. This one is just a template.

Reply:
If I understand correctly, @TerryGeng, everything inside the `<tr class="playlist-item-template d-none">` tag is just a template, right? If so, I can fix this issue easily: I can set the `<tr>` tag to be hidden from view for blind users.

Reply:
And that would not need the templating code to change, because I would change the attributes of the container, and not of any of the templates. If that makes sense.

Reply:
Okay I added an `aria-hidden="true"` attribute to the parent as well.